site/docs/numbers/lookupNumbers.mdx

```diff
@@ -328,3 +353,9 @@ Content-Type: application/json
 
 </TabItem>
 </Tabs>
+
+
+## Where to next?
+Now that you have learned how to lookup for numbers, check out some of the other available actions in our guides:
+- [How to port numbers](/docs/numbers/guides/porting/portingNumbers)
```

Comment:
I would suggest leaving out the link to the port numbers guide.  Customers may think that this is used in that workflow when it should not be.